eng/ignore-links.txt

```diff
@@ -16,5 +16,10 @@ https://docs.microsoft.com/javascript/api/@azure/arm-extendedlocation
 https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/quota/arm-extendedlocation
 https://docs.microsoft.com/javascript/api/@azure/arm-videoanalyzer
 https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/videoanalyzer/arm-videoanalyzer
+<<<<<<< HEAD
```

Comment:
Please resolve the merge conflict here

Reply:
Also we should try to avoid adding anything to this ignore-links file.